applications/fishing-map/src/features/search/Search.tsx

```diff
@@ -420,12 +421,16 @@ function Search() {
                                 </div>
                               )}
                               {firstTransmissionDate && lastTransmissionDate && (
-                                <div className={styles.property}>
+                                <div className={cx(styles.property, styles.fullWidth)}>
                                   <label>{t('vessel.transmission_plural', 'Transmissions')}</label>
                                   <span>
                                     from <I18nDate date={firstTransmissionDate} /> to{' '}
                                     <I18nDate date={lastTransmissionDate} />
                                   </span>
+                                  <TransmissionsTimeline
```

Comment:
There are some datasets with no transmission data... so I'd not render the component if the data is not available to avoid the empty timeline:
<img width="356" alt="image" src="https://user-images.githubusercontent.com/10500650/130599862-d69385a5-8a40-45bd-aaa8-01df3c746f0c.png">